install

```diff
@@ -89,6 +94,9 @@ while true; do
     --hip-clang)
         build_hip_clang=true
         shift ;;
+    --sanitizer)
```

Comment:
If using sanitizer does it need to be built in debug? I.e. should build_release=false be set when using --sanitizer?

Reply:
Yes it makes sense to build in debug mode, however I'm not sure if it is required